udata/tests/api/test_activities_api.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+from flask import url_for
+
+from udata.core.activity.models import Activity
+from udata.core.dataset.factories import DatasetFactory
+from udata.core.dataset.models import Dataset
+from udata.core.reuse.factories import ReuseFactory
+from udata.core.reuse.models import Reuse
+from udata.core.user.factories import UserFactory
+from udata.mongo import db
+from udata.tests.helpers import assert200
+
+pytestmark = [
+    pytest.mark.usefixtures("clean_db"),
+]
+
+
+class FakeDatasetActivity(Activity):
+    key = "fakeDataset"
+    related_to = db.ReferenceField(Dataset, required=True)
+
+
+class FakeReuseActivity(Activity):
+    key = "fakeReuse"
+    related_to = db.ReferenceField(Reuse, required=True)
+
+
+class ActivityAPITest:
+    modules = []
+
+    def test_activity_api_list(self, api):
```

Comment:
Just for my personal culture, you decided not to type hint the tests functions? I'm fine with either option, just so I can comply to some team rule :)

Reply:
I just forgot, thanks a lot for the reminder! 🙏